plugins/generateRss.js

```diff
@@ -0,0 +1,67 @@
+import fs from 'fs'
+import path from 'path'
+import Prismic from 'prismic-javascript'
```

Comment:
this `prismic-javascript` library has been deprecated for some time now. Please use `@prismicio/client` which is buried within the `@prismicio/vue` package. 

Here's a quick example of usage:
```
import * as prismic from '@prismicio/client'

const client = prismic.client(prismicApiUrl)
const posts = await client.query([
  prismic.predicates.at('document.type', 'post')
], {
  orderings: '[my.post.date desc]',
  pageSize: 10
})
```

Reply:
I had issues with that lib, and had downgraded to `prismic-javascript` to be with the times of nuxt and other dependencies (that are also outdated or deprecated) currently in the package.json.

This time I did not install `@prismicio/client`, and used the version bundled with `@prismicio/vue`, and it worked this time.

Whew, nice catch! 🎉🎉🎉